src/tls_stream.rs

```diff
@@ -56,25 +59,49 @@ impl Builder {
         self
     }
 
-    #[allow(dead_code)]
-    /* pub */ fn cert_store(&mut self, cert_store: CertStore) -> &mut Builder {
+    /// Specifies a custom certificate store which is later used when validating
+    /// a server's certificate.
+    ///
+    /// This option is only used for client connections and is used to construct
+    /// the certificate chain which the server's certificate is validated
+    /// against.
+    pub fn cert_store(&mut self, cert_store: CertStore) -> &mut Builder {
         self.cert_store = Some(cert_store);
         self
     }
 
+    /// Indicates whether this builder will be accepting a connection or not.
+    ///
+    /// For servers this flag should be set to `true`, and for clients no action
+    /// need be taken as this option is `false` by default.
+    pub fn accept(&mut self, accept: bool) -> &mut Builder {
```

Comment:
I'd rather make this another "terminal" method like `initialize`, or store this state in the `SchannelCred` and figure it out for the user.